task/backend/coordinator.go

```diff
@@ -61,3 +61,53 @@ func NotifyCoordinatorOfExisting(ctx context.Context, ts TaskService, coord Coor
 
 	return nil
 }
+
+type TaskResumer func(ctx context.Context, id influxdb.ID, runID influxdb.ID) error
+
+// TaskNotifyCoordinatorOfExisting lists all tasks by the provided task service and for
+// each task it calls the provided coordinators task created method
+// TODO(docmerlin): this is temporary untill the executor queue is persistent
+func TaskNotifyCoordinatorOfExisting(ctx context.Context, ts TaskService, tcs TaskControlService, coord Coordinator, exec TaskResumer, logger *zap.Logger) error {
```

Comment:
It sounds like this is outstanding work that needs to be completed before the scheduler for OSS is ready?

It appears we are making a decision to ignore all prior task runs when the `influxd` service is restarted. I would expect the default behavior should run all tasks. A reasonable option is to add a command line flag to specify the value of `latestCompleted`, so the user can choose how far back to run tasks to catch up.